DropDownBox: add a Getting Started tutorial (#4159)
DevExpress/devextreme-documentation: concepts/05 UI Components/DropDownBox/00 Getting Started with DropDownBox/15 Configure Selection.md

> The previous topic shows how to embed the List component into the DropDownBox's drop-down field. To implement a scenario where selected in the List values appear in the DropDownBox's text field, do the following:

Reviewer: > the DropDownBox's

This reads as DropDownBoks's - that's why you need to avoid such scenarios.

> where selected in the List values appear

Wrong word order.

```suggestion
The previous topic shows how to embed the List component into the DropDownBox component's drop-down field. To implement a scenario where the values selected in the List appear in the component's text field, do the following:
```

> 1. Bind the same data source to the List and DropDownBox. This tutorial uses a basic array as a data source.

Reviewer: ```suggestion
1. Bind the same data source to the List and DropDownBox components. This tutorial uses a basic array as a data source.
```

> 2. Specify the [selectionMode](/Documentation/ApiReference/UI_Components/dxList/Configuration/#selectionMode) property in the List that enables selection.

Reviewer: Who enables selection? The property or your list? It's unclear.

Author: I will rewrite this sentence.

> 3. Implement the [onSelectionChanged](/Documentation/ApiReference/UI_Components/dxList/Configuration/#onSelectionChanged) event handler. In this handler, pass the selected value to the DropDownBox [value](/Documentation/ApiReference/UI_Components/dxDropDownBox/Configuration/#value) option.

Reviewer: Did I get you right?

```suggestion
3. Implement the [onSelectionChanged](/Documentation/ApiReference/UI_Components/dxList/Configuration/#onSelectionChanged) event handler. In this handler, pass the selected value to the DropDownBox component's [value](/Documentation/ApiReference/UI_Components/dxDropDownBox/Configuration/#value) option.
```

Author: Yes, you got it right.

> If you want to implement different scenarios, like bind the DropDownBox and the embedded element to different data sources or more complex data sources, or select the initial value, refer to the following article: [Synchronize with the Embedded Element](/Documentation/Guide/UI_Components/DropDownBox/Synchronize_with_the_Embedded_Element/)

Reviewer: > like bind the DropDownBox and the embedded element to different data sources or more complex data sources

Something is amiss here. Please try to find it yourself.

Author: I rewrote this section
